spec/octokit/client/apps_spec.rb

```diff
@@ -60,6 +60,15 @@
 
       assert_requested request
     end
+
+    it "utilizes auto_pagination", :vcr do
```

Comment:
I'm wondering if `allows` would be clearer than `utilizes`. I'd be afraid that the endpoint automatically is auto-paginated, whereas this requires the user to set `auto_paginate` to true.